script/microsoft_tts.py

```diff
@@ -0,0 +1,25 @@
+"""Helper script to update supported languages for Microsoft Text-to-Speech (TTS)."""
+from pathlib import Path
+
+from lxml import html
+import requests
+
+from .hassfest.serializer import format_python_namespace
+
+URL = "https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/language-support"
+XPATH_QUERY = "//section[@data-tab='tts']/table[1]/tbody/tr/td[1]/code/text()"
```

Comment:
What if this page changes layout?

Reply:
I think that's fine, the script is run manually, and is meant to make it easier to regenerate the list of supported languages as I understand it.